### PR TITLE
Feature session and private routing

### PR DESCRIPTION
**Pull request by ema999**
## Summary

Adding the react router featuring and Navbar component

## Screenshots

Peek 2020-11-13 08-50

## Trello Card

https://trello.com/c/BPkcVTFN
https://trello.com/c/w4ypkKo4

### CODE REVIEW

File: src/components/NavBar/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import logo from '../../app/assets/logo.png';
+
+import styles from './styles.module.scss';
+
+function NavBar({ links }) {
+  return (
+    <div className={styles.navBar}>
+      <img src={logo} className={styles.logo} />
+      <div>
```
Comment (damfinkel) [inline src/components/NavBar/index.js]: ul

File: src/components/NavBar/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {links &&
+          links.map(link => (
+            <span
```
Comment (damfinkel) [inline src/components/NavBar/index.js]: li

File: src/components/NavBar/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@import 'variables/colors';
+
+.nav-bar {
+  align-content: stretch;
+  align-items: flex-start;
+  border-top: 4px solid $blue;
+  display: flex;
+  flex-direction: row;
```
Comment (damfinkel) [inline src/components/NavBar/styles.module.scss]: this is the default value, it's not necessary to explicityly set

File: src/components/PrivateRoute/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <Route
+      {...rest}
+      render={({ location }) =>
+        isLogged ? (
+          children
+        ) : (
+          <Redirect
+            to={{
+              pathname: '/login',
```
Comment (damfinkel) [inline src/components/PrivateRoute/index.js]: Define paths as constants somewhere, they'll probably be used somewhere else

File: src/components/PublicRoute/index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (damfinkel) [inline src/components/PublicRoute/index.js]: This component and PrivateRoute are too similar. Can't we somehow reuse the code from both?

File: src/components/PublicRoute/index.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+function PublicRoute({ children, ...rest }) {
+  const isLogged = isLoggedIn();
+  return (
+    <Route
+      {...rest}
+      render={({ location }) =>
+        isLogged ? (
+          <Redirect
+            to={{
+              pathname: '/',
```
Comment (damfinkel) [inline src/components/PublicRoute/index.js]: constant

**Review by damfinkel — CHANGES_REQUESTED** [review]

**Review by damfinkel — APPROVED** [review]

**Review by DavidRojas1612 — APPROVED** [review]

**Pull request merged**